test.py

```diff
@@ -0,0 +1,8 @@
+from dcolor import dc
+
+#dc.showPlot(lambda z : (z**5+1j*z**4+z**3+1j*z**2+z+1j+1)/np.sin(z))
+dc.savePlot(lambda z : z**3+2j*z**2+2j+z,"foo.jpg")
```

Comment:
`jpg` doesn't seem to be an accepted file format -- the accepted formats appear to be:

```
eps, pdf, pgf, png, ps, raw, rgba, svg, svgz
```